Lib/gzip.py

```diff
@@ -231,6 +253,11 @@ def _init_write(self, filename):
         self.bufsize = 0
```

Comment:
Should there be a comment to clarify the difference between `self.bufsize` and `self.buffer_size`?

Reply:
Looks like `self.bufsize` is actually not used anywhere, at least in the gzip module. Might be best to just delete it?